flow/connectors/utils/partition.go

```diff
@@ -128,6 +129,20 @@ func createUIntPartition(start uint64, end uint64) *protos.QRepPartition {
 	}
 }
 
+func createObjectIdPartition(start bson.ObjectID, end bson.ObjectID) *protos.QRepPartition {
+	return &protos.QRepPartition{
+		PartitionId: uuid.New().String(),
+		Range: &protos.PartitionRange{
+			Range: &protos.PartitionRange_ObjectIdRange{
+				ObjectIdRange: &protos.ObjectIdPartitionRange{
+					Start: start.Hex(),
```

Comment:
nit: would it be more efficient if stored as bytes?

Reply:
For object ID, string is more convenient for building the bson document for building query filter. It only supports either string or ObjectId(<string>), but not bytes. So using Hex() here for convenience.